include/flatbuffers/flatbuffers.h

```diff
@@ -876,7 +876,9 @@ FLATBUFFERS_FINAL_CLASS
   /// @param[in] str A const pointer to a `String` struct to add to the buffer.
   /// @return Returns the offset in the buffer where the string starts
   Offset<String> CreateString(const String *str) {
-    return CreateString(str->c_str(), str->Length());
+    if (str)
```

Comment:
auto str = builde.createString(flatBufferObject->someString());
if flatBufferObject->someString() is null the above line will crash.